Automattic/newspack-ads
# feat: centralize iab sizes

Unify IAB sizes and implement its official names.

Closes #256 

### How to test

1. Check out this branch and https://github.com/Automattic/newspack-plugin/pull/1577
2. Connected to GAM, edit an existing ad unit and confirm its sizes are preserved and names are applied for standard sizes
3. Create a new ad unit and confirm you are able to select different sizes and save
4. Confirm the new ad unit sizes are applied by visiting your GAM dashboard
